packages/playwright/spec/chai-extra.ts

```diff
@@ -0,0 +1,8 @@
+import * as chaiLib from 'chai';
```

Comment:
If you add a `devDependency` on `"@integration/testing-tools": "2.0.0"` to `packages/playwrigth/package.json` you can then grab an `expect` that's [already wired](https://github.com/serenity-js/serenity-js/blob/master/integration/testing-tools/src/expect/index.ts) with `sinon` and `chai-as-promised`:

```typescript
import { expect } from '@integration/testing-tools';
````

Also, would you be OK with using the `expect` API rather than `should` to keep the tests consistent across all the modules?

Reply:
Sure, will do!